src/js/cart.js

```diff
@@ -1,20 +1,35 @@
 import { getLocalStorage } from "./utils.mjs";
 
+// Function to calculate the final price after applying discount
+function calculateFinalPrice(item) {
+  const discountAmount = (item.FinalPrice * item.Discount) / 100;
+  return item.FinalPrice - discountAmount;
+}
+
+// Function to render the cart contents
 function renderCartContents() {
   const cartItems = getLocalStorage("so-cart");
 
+<<<<<<< HEAD
   // Check if there are any items in the cart and if it's an array
   if (!Array.isArray(cartItems) || cartItems.length === 0) {
     document.querySelector(".product-list").innerHTML =
       "<p>Your cart is empty.</p>";
     return;
   }
 
+  // Map through the cart items and generate HTML for each
+=======
+>>>>>>> 38118cfeb7a6d88203c71cd874958f4176df1e4f
```

Comment:
@nengiamoni Make sure you remove these merge conflict lines or else you'll get errors.

Reply:
i think have gone to scatter the code now